app/src/main/java/io/homeassistant/companion/android/sensors/BluetoothSensorManager.kt

```diff
@@ -353,27 +352,22 @@ class BluetoothSensorManager : SensorManager {
 
         val icon = if (monitoringManager.isMonitoring()) "mdi:bluetooth" else "mdi:bluetooth-off"
 
-        var state = if (!BluetoothUtils.isOn(context)) "Bluetooth is turned off" else if (monitoringManager.isMonitoring()) "Monitoring" else "Stopped"
+        val state = if (!BluetoothUtils.isOn(context)) "Bluetooth is turned off" else if (monitoringManager.isMonitoring()) "Monitoring" else "Stopped"
 
-        var attr: Map<String, Any?> = mapOf()
+        val attr: MutableMap<String, Any?> = mutableMapOf()
```

Comment:
Try using type inference if possible.

```suggestion
        val attr = mutableMapOf<String, Any?>()
```

Reply:
will add in my next commit, thank you 🙏